packages/mlflow/scripts/deploy_kind_cluster.sh

```
@@ -33,15 +33,15 @@ popd > /dev/null
 
 echo "Deploying MLflow UI..."
-kustomize edit set namespace kubeflow
-kubectl apply -n kubeflow -k .
+kustomize edit set namespace mlflow
+kubectl apply -n mlflow -k .
 
 # Wait for deployment to be available
 echo "Waiting MLflow UI to be available..."
-kubectl wait --for=condition=available -n kubeflow deployment/mlflow-ui --timeout=1m
+kubectl wait --for=condition=available -n mlflow deployment/mlflow-ui --timeout=1m
 popd > /dev/null
```

Shellcheck (0.11.0)

[warning] Use 'popd ... || exit' or 'popd ... || return' in case popd fails.

(SC2164)